packages/plugin-core/src/constants.ts

```diff
@@ -840,6 +840,17 @@ export const DENDRON_COMMANDS: { [key: string]: CommandEntry } = {
     },
     when: "dendron:pluginActive",
   },
+  TOGGLE_PREVIEW: {
+    key: "dendron.togglePreview",
+    title: `${CMD_PREFIX} Toggle Preview`,
+    icon: `$(open-preview)`,
+    // keybindings: {
```

Comment:
can we replace the show preview command key bindings with `togglePreview`? this should be the default going forward

Reply:
so regardless if the user invokes `ShowPreview` or `TogglePreview`, the logic for `TogglePreview` should be executed?

Reply:
so regardless if the user invokes `ShowPreview` or `TogglePreview`, the logic for `TogglePreview` should be executed?

currently, I'm passing `isShowCommand` flag to the command to accommodate the old behavior when user invokes `ShowToggle`. If the above is the case, I can just get rid of the flag and always execute toggle logic.

Reply:
except I only need the flag for showing deprecation message when `ShowToggle` is used

Reply:
I think I still need to maintain show command's key bindings for the above reason. I need to pass a flag to show deprecation message.

If our intention is to execute toggle logic for both `ShowPreview` and `TogglePreview`, I can change code inside `execute` instead of changing the key bindings.

Reply:
we'll execute toggle logic for both. the reason to show the deprecation message is if the user set a custom keyboard shortcut for `show toggle` that will stop working once we remove the command